pkg/agent/plugin/svidstore/utils.go

```diff
@@ -0,0 +1,94 @@
+package svidstore
+
+import (
+	"crypto/x509"
+	"fmt"
+	"strings"
+
+	"github.com/spiffe/spire/pkg/common/pemutil"
+	"github.com/spiffe/spire/pkg/common/x509util"
+	svidstorev1 "github.com/spiffe/spire/proto/spire/plugin/agent/svidstore/v1"
+)
+
+type Data struct {
```

Comment:
nit: The documentation on these fields isn't standard Godoc style (i.e. comments start with the name of the field)